src/docs/backup.dox

```diff
@@ -89,7 +89,74 @@ rm -rf /path/database.backup &&
     wt -h /path/database.source backup /path/database.backup
 @endcode
 
-@section backup_incremental Incremental backup
+@section backup_incremental-block Block-based Incremental backup
+
+Once a full backup has been done, it can be rolled forward incrementally by
+copying only modified blocks and new files to the backup copy directory.
+The application is responsible for retaining and removing files that
+are no longer part of the backup when later incremental backups no longer
+return their name. This is especially important for WiredTiger log files
+that are no longer needed and must be removed before recovery is run.
+
+@copydoc doc_bulk_durability
+
+The following is the procedure for incrementally backing up a database
+using block modifications:
+
+1. Perform a full backup of the database (as described above), with the
+additional configuration \c incremental=(enabled=true,this_id=”ID1”).
+The identifier specified in \c this_id starts block tracking and that
+identifier can be used in the future as the source of an incremental
+backup.
+
+2. Begin the incremental backup by opening a backup cursor with the
+\c backup: URI and config string of \c incremental=(src_id="ID1",this_id="ID2").
+Call this \c backup_cursor. Like a normal full backup cursor,
+this cursor will return the filename as the key.  There is no associated
+value.  The information returned will be based on blocks tracked since the time of
+the previous backup designated with "ID1".  New block tracking will be started as
+"ID2" as well.  WiredTiger will maintain modifications from two IDs, the current
+and the most recent completed one.
+
+3. For each file returned by \c backup_cursor->next(), open a duplicate
+backup cursor to do the incremental backup on that file.  The list
+returned will also include log files, \c WiredTigerLog, that need to
```

Comment:
I'd change: `log files, \c WiredTigerLog` to `log files (prefixed by \c WiredTigerLog),`